src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-prometheus.mdx

```diff
@@ -15,29 +16,22 @@ You can collect metrics from any [software that exposes Prometheus metrics](http
 
 Complete the steps below to collect Prometheus metrics.
 
-## Step 1: Sign up for your free account if you haven't already [#signup]
-
-<InlineSignup />
-
-## Step 2: Prerequisites [#prerequisites]
-
-The collector will use the Prometheus receiver component.
-
-Depending on the software you want to monitor the prerequites may change. Prometheus metrics endpoint could be enabled by defaut or may need some setup in the configuration to enable it.
-Others software can require a [third-party Prometheus exporter](https://prometheus.io/docs/instrumenting/exporters/#third-party-exporters) to expose metrics.
+<Steps>
+    <Step>
+    ## Ensure you have the prerequisites 
 
-In both cases, the Prometheus metrics endpoint needs to be accesible from the OpenTelemetry Collector.
+If you haven't already:
 
-## Step 3: Install Opentelemetry collector [#install-opentelemetry-collector]
+* [Sign up for a New Relic account](https://newrelic.com/signup).
+* In your target app, make sure you're exposing the Prometheus metrics endpoint so it is accessible to the Prometheus receiver component in the OpenTelemetry collector.
+  * The Prometheus metrics endpoint could be enabled by default or you may need do do additional configuration to enable it.
+  * Some software may require a [third-party Prometheus exporter](https://prometheus.io/docs/instrumenting/exporters/#third-party-exporters) to expose metrics.
+* Install the OpenTelemetry collector by following the steps in the [OpenTelemetry docs](https://opentelemetry.io/docs/collector/getting-started/). In some cases but not all, you may use an open source project that requires you to install the OpenTelemetry contrib distro. For example, if you use the Core distribution, the contrib distro contains additional components that the collector needs. 
+* Regardless of your distribution, you must have the [OTLP exporter](https://pkg.go.dev/go.opentelemetry.io/collector/exporter/otlpexporter) and the [Prometheus receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/prometheusreceiver) components installed.
```

Comment:
this point is related to the previous one, but it is confusing as it is now.

Moreover, I would not use the word "installed"